Altis_Life.Altis/core/init.sqf

```diff
@@ -42,24 +43,14 @@ diag_log "::Life Client:: Received server functions.";
 0 cutFadeOut 99999999;
 
 diag_log "::Life Client:: Waiting for the server to be ready..";
-waitUntil{!isNil "life_HC_isActive"};
-if (life_HC_isActive) then {
-    waitUntil{!isNil "life_HC_server_isReady" && !isNil "life_HC_server_extDB_notLoaded"};
-    _server_isReady = life_HC_server_isReady;
-    _extDB_notLoaded = life_HC_server_extDB_notLoaded;
-} else {
-    waitUntil{!isNil "life_server_isReady" && !isNil "life_server_extDB_notLoaded"};
-    _server_isReady = life_server_isReady;
-    _extDB_notLoaded = life_server_extDB_notLoaded;
-};
+waitUntil {!isNil "life_server_isReady" && !isNil "life_HC_isActive" && !isNil "life_server_extDB_notLoaded"};
```

Comment:
Can you do [lazy evaluation](https://community.bistudio.com/wiki/Code_Optimisation) on the latter two conditions?